compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/ParagraphLayouter.skiko.kt

```diff
@@ -219,6 +219,11 @@ internal class ParagraphLayouter(
     }
 }
 
-private fun Float.sameValueAs(other: Float) : Boolean {
-    return abs(this - other) < 0.00001f
-}
\ No newline at end of file
+private fun Shadow?.sameValueAs(other: Shadow?) =
```

Comment:
What if instead of checking on `null` and `None` everywhere, we just pass `None` instead of `null`? For example, here:
```
 override val textDecoration: TextDecoration? = null,
```
we can write:
```
 override val textDecoration: TextDecoration = TextDecoration.None,
```

This will reduce mental load on developers who read the code.